src/pages/List.js

```diff
@@ -3,11 +3,18 @@ import ListGroup from '../components/ListGroup';
 import AddGroupForm from '../components/AddGroupForm'
 import '../styles/App.css'
 import GroupsFilter from '../components/GroupsFilter';
+import MyModal from '../components/ui/modal/MyModal';
+import MyButton from '../components/ui/button/MyButton';
 
-const List = ({sortedAndSearch, groups, setGroups, addGroup, removeGroup, addCompleted, addNotCompleted, setLocalGroups, filter, setFilter}) => {
+const List = ({sortedAndSearch, groups, setGroups, addGroup, removeGroup, addCompleted, addNotCompleted, setLocalGroups, filter, setFilter, modalGroupVisible, setModalGroupVisible}) => {
```

Comment:
1. Maybe, it will be better for readability to change and add everywhere `props.`:
```suggestion
const List = (props) => {
```
OR at least:
```suggestion
const List = ({
sortedAndSearch, 
groups,
setGroups,
addGroup,
removeGroup,
addCompleted,
addNotCompleted,
setLocalGroups,
filter,
setFilter,
modalGroupVisible,
setModalGroupVisible
}) => {
```